products/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+
+from core.models import TimeStampModel
+
+
+class Categories(models.Model): 
+    name = models.CharField(max_length=50)
+
+
+class SubCategories(models.Model): 
+    name     = models.CharField(max_length=50)
+    category = models.ForeignKey('Categories', on_delete=models.CASCADE)
+
+
+class Products(TimeStampModel):
+    name         = models.CharField(max_length=150, unique=True)
+    eng_name     = models.CharField(max_length=250, unique=True)
+    rosting_date = models.DateField()
+    price        = models.DecimalField()
```

Comment:
```suggestion
    price        = models.DecimalField(max_digits = , decimal_places = )
```
[공식문서](https://docs.djangoproject.com/en/4.0/ref/models/fields/#django.db.models.ForeignKey)를 참고하여 DecialField를 프로젝트 기획에 맞게 적절하게 수정해주세요.